### PR TITLE
TASK: Improve content dimensions documentation

### PR DESCRIPTION
**Pull request by aertmann**

### CODE REVIEW

File: TYPO3.Neos/Documentation/CreatingASite/ContentDimensions.rst [hunk 1]
```diff
@@ -184,6 +184,20 @@ example:
...
+=============================
+
+Adjusting content dimensions configuration can lead to issues for existing content. When a new content dimension is added,
+a corresponding value needs to be added to existing content, otherwise no nodes would be found.
+
+This can be done with a node migration which is included in the ``TYPO3.TYPO3CR`` package::
+
+	./flow node:migrate 20150716212459
+
+This migration adds missing content dimensions by setting the default value on all existing nodes, if not already set.
```
Comment (gerhard-boden) [inline TYPO3.Neos/Documentation/CreatingASite/ContentDimensions.rst]: flexibility

File: TYPO3.Neos/Documentation/CreatingASite/ContentDimensions.rst [hunk 2]
```diff
@@ -184,6 +184,20 @@ example:
...
+
+Alternatively a custom node migration can be created allowing flexibilty and constraints. See :ref:`node-migrations`.
```
Comment (gerhard-boden) [inline TYPO3.Neos/Documentation/CreatingASite/ContentDimensions.rst]: flexibility ;)

**Comment by gerhard-boden** [comment]
too tiny typos otherwise good to go! :+1:

**Comment by aertmann** [comment]
Thanks :+1: Fixed typo, not sure why you say there are two though?

**Comment by gerhard-boden** [comment]
for some reason I thought I saw two, pretty sure that I'm sober :smile:

**Comment by kdambekalns** [comment]
Are you sure you use the migration you want to use? 20150716212459 migrates
> from some node dimensions to default dimension values, adding missing dimension default values.

whereas 20140326143834 migrates

> from no node dimensions to default dimension values.

The 2015 migration is for those that add _another_ dimension, the 2014 migration is for those adding one or more dimension without having any.

**Comment by aertmann** [comment]
@kdambekalns: Heh yes I'm sure, otherwise I wouldn't push such a change. In practice the existing one is rather useless unless you don't have any existing dimensions. However the new one works in both cases.

**Comment by aertmann** [comment]
keep in mind that it doesn't overwrite the existing dimension values if they already exist.. maybe it should be better described too

**Comment by aertmann** [comment]
I'm pretty sure less people will have less trouble using dimensions than the existing documentation.

**Pull request merged**